src/swiftrax/cowfish.json

```diff
@@ -0,0 +1,255 @@
+{
+  "name": "CowFish",
+  "vendorId": "0x04D8",
+  "productId": "0xEB53",

```

Comment:
This doesn't match the QMK PR:
```#define VENDOR_ID       0x7161
#define PRODUCT_ID      0x5239```

Reply:
Yeah sorry opened a new PR changing VID and PID
https://github.com/qmk/qmk_firmware/pull/9922